pkg/subctl/cmd/join.go

```diff
@@ -458,6 +462,18 @@ func populateSubmarinerSpec(subctlData *datafile.SubctlData, netconfig globalnet
 	if len(customDomains) > 0 {
 		submarinerSpec.CustomDomains = customDomains
 	}
+	if len(imageOverrideArr) > 0 {
+		imageOverrides = make(map[string]string)
+		for _, s := range imageOverrideArr {
+			key := strings.Split(s, "=")[0]
+			value := strings.Split(s, "=")[1]
+			imageOverrides[key] = value
+		}
+	} else {
+		imageOverrides = nil
```

Comment:
Pretty sure it's nil by default, is this necessary?

Reply:
I'm not sure because I'm using here another variable `imageOverrideArr` to store the value from the command flag which always has a **nil** default but the other variable `imageOverrides` is just the conversion from `[]string` to `map[string]string`. I don't see how it gets **nil** as well.

Reply:
@SteveMattar @mkolesnik is right, it's an uninitialized variable, which for go is "nil" by default. Golang zeroes-out all variables (this is a difference with C/C++)

Reply:
ACK! Thanks guys!